tio/command_parser.c

```diff
@@ -5,6 +5,10 @@
 
 #include <stdlib.h>
 
+static void _trace_op(char* op, int res) {
```

Comment:
I think we can use `#ifdef TIO_TRACE` here to avoid adding it in each place (or make `_trace_op` a macro